### ISSUE TITLE
DATAFRAME UNABLE TO OPEN

### ISSUE THREAD

**Issue by sourabh2831** [spyder-ide/spyder#20783]
### What steps will reproduce the problem?

def get_open_order():
    response=obj.orderBook()
    df=pd.DataFrame(response['data'])
    return df[df["orderstatus"]=="AMO CANCELLED"]
open_orders=get_open_order()

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Users\Chopras\.conda\envs\angelone\lib\site-packages\spyder\plugins\variableexplorer\widgets\collectionsdelegate.py", line 211, in createEditor
    if not editor.setup_and_check(value, title=key):
  File "C:\Users\Chopras\.conda\envs\angelone\lib\site-packages\spyder\plugins\variableexplorer\widgets\dataframeeditor.py", line 957, in setup_and_check
    self.dataModel = DataFrameModel(data, parent=self)
  File "C:\Users\Chopras\.conda\envs\angelone\lib\site-packages\spyder\plugins\variableexplorer\widgets\dataframeeditor.py", line 135, in __init__
    self.max_min_col_update()
  File "C:\Users\Chopras\.conda\envs\angelone\lib\site-packages\spyder\plugins\variableexplorer\widgets\dataframeeditor.py", line 244, in max_min_col_update
    for __, col in self.df.iteritems():
  File "C:\Users\Chopras\.conda\envs\angelone\lib\site-packages\pandas\core\generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'iteritems'
```

## Versions

* Spyder version: 5.4.2  (conda)
* Python version: 3.8.16 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.7
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.31.1;<9.0.0       :  8.10.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.17.3 (OK)
keyring >=17.0.0              :  23.13.1 (OK)
nbconvert >=4.0               :  6.5.4 (OK)
numpydoc >=0.6.0              :  1.5.0 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0;<3.0           :  2.16.2 (OK)
pylint_venv >=2.1.1           :  2.3.0 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.7.1;<1.8.0          :  1.7.1 (OK)
pylsp_black >=1.2.0           :  1.2.1 (OK)
qdarkstyle >=3.0.2;<3.1.0     :  3.0.2 (OK)
qstylizer >=0.2.2             :  0.2.2 (OK)
qtawesome >=1.2.1             :  1.2.2 (OK)
qtconsole >=5.4.0;<5.5.0      :  5.4.0 (OK)
qtpy >=2.1.0                  :  2.2.0 (OK)
rtree >=0.9.7                 :  1.0.1 (OK)
setuptools >=49.6.0           :  44.1.1 (NOK)
sphinx >=0.6.6                :  5.0.2 (OK)
spyder_kernels >=2.4.2;<2.5.0 :  2.4.2 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=22.1.0                  :  23.2.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=3.0.0            :  None (NOK)
numpy >=1.7                   :  1.24.2 (OK)
pandas >=1.1.1                :  2.0.0 (OK)
scipy >=0.17.0                :  None (NOK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by ccordoba12** [comment]
Hey @sourabh2831, thanks for reporting. This problem is fixed in our latest version (5.4.3), released a couple of days ago.

If you're using Conda-forge, then you can update to it by opening the Anaconda Prompt and running there

    conda activate angelone
    conda install -c conda-forge spyder=5.4.3

If you are not using Conda-forge, or you don't know what that is, you need to downgrade Pandas to its 1.5 version to avoid this problem:

    conda install pandas=1.5

Let us know if that works for you.

**Comment by sourabh2831** [comment]
Hi,

I did try downgrading pandas to 1.5 but it still didn't work.

Br,
Sourabh

**Comment by ccordoba12** [comment]
I can assure you the problem is fixed with Pandas 1.5. So, you need to be sure to introduce the above commands correctly.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
